clear/main.py

```diff
@@ -550,6 +551,25 @@ def route(prefix, vrf, namespace):
 helper = util_base.UtilHelper()
 helper.load_and_register_plugins(plugins, cli)
 
+@click.option('--all', '-a', is_flag=True, help='Delete also compressed logs')
+@cli.command()
+def logging(all):
+    """Clear logging files"""
+    if os.path.exists("/var/log.tmpfs"):
+        log_path = "/var/log.tmpfs"
+    else:
+        log_path = "/var/log"
+    
+    if all:
+        files_to_delete = glob.glob("{}/syslog*".format(log_path))
+    else:
+        files_to_delete = glob.glob("{}/syslog".format(log_path))
+    if os.path.isfile("{}/syslog.1".format(log_path))
```

Comment:
Why check for `syslog.1` specifically if not clearing `--all` logs?
I would remove this line. If no `--all` option is requested, then only remove the current logs. Otherwise remove all backup logs as well, including `syslog.1`

Reply:
this is copying what is being done in the show logging

Reply:
@matiAlfaro is right, if he doesn't delete it, the show logging command will take the syslog.1